source/probability_theory_4_finite.Rmd

```diff
@@ -1047,10 +916,75 @@ Convert to a proportion.
 
 Print the results.
 
-Result: kk = .01
+```{python, opts.label="py_ed"}
+N = 10000
+trial_results = np.zeros(N, dtype=bool)
+
+# Deck with 13 spades (ones) and 39 other cards (zeros)
+deck = np.concatenate([
```

Comment:
`np.append` easier to explain?

Reply:
Except that `np.append` is evil, and Shall Never Be Used.

Reply:
Evil because it can be used for single elements, and that gets ugly?   Any evil implied from simple concatenate arrays usage here?

Reply:
No, but I prefer not teaching techniques which will be repeated later to ill effect.  I don't think `np.append` is ever really useful.